libdebug/ptrace/ptrace_interface.py

```diff
@@ -13,9 +13,6 @@
 from pathlib import Path
 from typing import TYPE_CHECKING
 
```

Comment:
_peek_user and _poke_user are not used anymore

Reply:
We might need that dead code in the future, you never know...

Reply:
You know that I do not agree with this policy o.o